server/src/main/java/io/seata/server/lock/AbstractLockManager.java

```diff
@@ -153,6 +153,9 @@ protected Locker getLocker() {
                                             Long branchID) {
         List<RowLock> locks = new ArrayList<RowLock>();
 
+        if (StringUtils.isEmpty(lockKey)) {
```

Comment:
That s ok. And I think can remove the code
```
if (StringUtils.isNullOrEmpty(lockKey)) {
            // no lock
            return true;
}
```
at 
```
acquireLock(BranchSession branchSession)
```

Reply:
`isLockable `will call this also，and it has no NEP judgement;